cluster/service/cluster_service_whitebox_test.go

```diff
@@ -145,7 +144,7 @@ func TestValidation(t *testing.T) {
 				// then
 				require.Error(t, err)
 				require.IsType(t, errors.BadParameterError{}, err)
-				assert.Equal(t, fmt.Sprintf(errInvalidURLMsg, "API", c.URL), err.(errors.BadParameterError).Error())
+				assert.Equal(t, "'API' URL '' is invalid: missing scheme or host", err.(errors.BadParameterError).Error())
```

Comment:
Why still not using https://github.com/fabric8-services/fabric8-common/blob/b10e057d860d730661b2440dc1326c7d82606acc/test/error.go#L11 here and in other places? You could use just one statement instead of three asserts/requires (error not nil, type, mesage). See an example of usage here - https://github.com/fabric8-services/fabric8-common/blob/b10e057d860d730661b2440dc1326c7d82606acc/auth/service_blackbox_test.go#L70

Reply:
ok, let me do that

Reply:
addressed in af4ee21